lib/web/apiserver_test_utils.go

```diff
@@ -29,10 +29,14 @@ import (
 )
 
 // NewDebugFileSystem returns the HTTP file system implementation
-func newDebugFileSystem() (http.FileSystem, error) {
+func NewDebugFileSystem(isEnterprise bool) (http.FileSystem, error) {
```

Comment:
Does it need to be exported?

Reply:
i forgot to push the branch that required this change in enterprise: https://github.com/gravitational/teleport.e/pull/5818, i added a enterprise not found handler test there (just in case...)